makeflow/src/makeflow.c

```diff
@@ -1152,6 +1216,9 @@ int main(int argc, char *argv[])
 		{"help", no_argument, 0, 'h'},
 		{"makeflow-log", required_argument, 0, 'l'},
 		{"max-local", required_argument, 0, 'j'},
+		{"-max_local_disk", required_argument, 0, '1'},
```

Comment:
You will want to make this correspond to a LONG_OPT instead of 1,2,3 to avoid confusion. You can also drop the '-' at the beginning of each as all long options have double hyphen leading automatically.